apps/system/js/wrapper_factory.js

```diff
@@ -182,8 +191,10 @@
 
     generateBrowserConfig: function wf_generateBrowserConfig(features) {
       var config = {};
+      config.chrome = {};
       config.title = features.name;
       config.icon = features.icon || '';
+      config.chrome.pinned = features.pinned || false;
```

Comment:
Is there a particular reason we have to set this? I'm not sure it's a good idea to set it if it never gets updated. A window can transform between pinned and unpinned state during its lifetime, we should track this state on the AppChrome object rather than set it in config for the lifetime of the window.